packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -17,6 +17,7 @@ export { NodeHttpHandlerOptions };
 interface ResolvedNodeHttpHandlerConfig {
   requestTimeout?: number;
   connectionTimeout?: number;
+  socketAcquisitionWarningTimeout?: number;
```

Comment:
Can we mark this as `@internal` for now?

This is more for debugging rather than something customers should actually configure.

Reply:
This is intended to be configured by the customer. Because this is a console warning, there should be a way to shut it off. Setting a high timeout here would do it.

We don't use this number in an "internal" way. It goes straight from this config to the timeout function.